mentoring/components/templates/html/mcqblock.html

```diff
@@ -0,0 +1,19 @@
+<fieldset class="choices questionnaire">
+  <legend class="question">
+    {% if not hide_header %}<h3 class="question-title">{{ self.display_name }}</h3>{% endif %}
+    <p>{{ self.question }}</p>
+  </legend>
+  <div class="choices-list">
+    {% for choice in custom_choices %}
+    <div class="choice">
+      <div class="choice-result fa icon-2x"></div>
```

Comment:
Also, selecting "No" gives a correct result - it should be wrong as per the Studio config above.

![screenshot-sandbox opencraft com 2015-02-24 14-35-17](https://cloud.githubusercontent.com/assets/514483/6350255/661ce2a2-bc32-11e4-81a7-15be67387565.png)

http://sandbox.opencraft.com/courses/course-v1:TestX+TST-MENTORING-2+now/courseware/f38b4f3ba1c140aeb14daaba27401211/4e69426ce8a149bebcc1fe83b64a6475/1